### ISSUE TITLE
Microphone is visible but does nothing

### ISSUE THREAD

**Issue by MortimerGoro** [MozillaReality/FirefoxReality#204]
Microphone is visible in the URL bar and keyboard but does nothing (we haven't implemented the functionality yet).

Should we hide it for V1?

**Comment by avrignaud** [comment]
Not currently. We are going to use the visibility of the microphone button
in the FxR User Research study we're doing next week to see what people
volunteer. However, we should discuss whether or not to remove the
microphone for the 1.0 launch to reduce confusion.

**Comment by cvan** [comment]
@avrignaud: What have the User Research studies concluded? I think we should we remove the microphone for now, personally.

**Comment by thenadj** [comment]
Hoping to get minimal voice in before shipping. Let's leave it for now. Users definitely want voice!

**Comment by avrignaud** [comment]
Yep, voice search is important, voice command less so but still interesting to help mitigate challenges around keyboard text entry. We are trying to get one or both in for 1.0 using Mozilla's Open Voice work, and the microphone will likely be one of the triggers.

**Comment by andrenatal** [comment]
I'm currently working to integrate the speech API and make the microphone trigger it.
